tests/acceptance/features/apiGraphUserGroup/deleteUser.feature

```diff
@@ -166,3 +166,15 @@ Feature: delete user
     When the user "Alice" deletes a user "Brian" using the Graph API
     Then the HTTP status code should be "204"
     And as "Alice" folder "Shares/new" should not exist
+
+  @issue-11027
+  Scenario: admin user deletes a user containing + character in the username
+    Given the administrator has assigned the role "Admin" to user "Alice" using the Graph API
```

Comment:
please add it to the existing test scenario mentioned in https://github.com/owncloud/ocis/issues/11032